CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ endif()
 set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE CACHE BOOL "Add paths to linker search and installed rpath")
 
 # AMD targets
-set(AMDGPU_TARGETS gfx900:xnack-;gfx906:xnack-;gfx908:xnack- CACHE STRING "List of specific machine types for library to target")
+set(AMDGPU_TARGETS gfx900;gfx906;gfx908 CACHE STRING "List of specific machine types for library to target")
```

Comment:
Need to leave the xnack flags on

Reply:
If we leave the xnack, I am getting the followinf error on rocm3.9
```
Invalid offload arch combinations: gfx906 and gfx906:xnack- 
```

Reply:
Please see https://github.com/ROCmSoftwarePlatform/rocRAND/pull/158